### ISSUE TITLE
Use a test set

### ISSUE THREAD

**Issue by tupini07** [tupini07/StyleTransfer#7]
Currently loss is evaluated only on training set
